README.md

```diff
@@ -1,5 +1,33 @@
 # curriculum-cr-training-contact-form
+# __Hi there 👋:heart: !__
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+## Portfolio Project
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+## About
+
+This is my personal website showcasing my work as a software developer. In this portifolio, you will see projects I have worked on and skills I have acquired.
```

Comment:
Please, add more information about the project, it's gonna be useful for you to explain with more details about your capabilities for future recruiters.

- [ ] Improve README project description.
- [ ] Add acknowledgments and reference the Figma Design again.